internal/build/lifecycle_execution.go

```diff
@@ -228,10 +235,24 @@ func (l *LifecycleExecution) Run(ctx context.Context, phaseFactoryCreator PhaseF
 
 		group, _ := errgroup.WithContext(context.TODO())
 		if l.platformAPI.AtLeast("0.10") && l.hasExtensionsForBuild() {
-			group.Go(func() error {
-				l.logger.Info(style.Step("EXTENDING (BUILD)"))
-				return l.ExtendBuild(ctx, buildCache, phaseFactory)
-			})
+			if false {
```

Comment:
Can we leave a code comment here that we are planning to deprecate the older method?

Reply:
added earlier!